ext/posix/posix.stub.php

```diff
@@ -429,3 +429,4 @@ function posix_initgroups(string $username, int $group_id): bool {}
 function posix_sysconf(int $conf_id): int {}
 
 function posix_pathconf(string $path, int $name): int|false {}
+function posix_fpathconf(int $fd, int $name): int|false {}
```

Comment:
Due to the fact `resource` is not a type you cannot use a proper type declaration here and will need to do it as a doc comment (a bit like how ``posix_isatty()`` is marked)